src/ElmSharp/ElmSharp/Widget.cs

```diff
@@ -294,6 +294,13 @@ public bool IsAutoMirroredMode
         public void SetFocus(bool isFocus)
         {
             Interop.Elementary.elm_object_focus_set(RealHandle, isFocus);
+
+            // Temporary code due to focus issue (will be removed at Tizen 5.0)
+            if (this is Window window)
```

Comment:
'window' is never used on the following block

Reply:
done